nodebuilder/state/flags.go

```diff
@@ -36,6 +38,13 @@ func Flags() *flag.FlagSet {
 		false,
 		"enables TLS for the estimator service gRPC connection",
 	)
+	flags.Int(
+		txWorkerAccountsFlag,
+		1,
+		"specifies how many accounts the TxClient manages for PayForBlob submission.\n"+
+			"0 disables parallel submission. Values greater than 1 automatically create "+
+			"\"parallel-worker-*\" accounts and grant them fees using the default signer.",
```

Comment:
nit: specify behaviour of when this is set to 1
```suggestion
		"is used for queued submission. It defines how many accounts the TxClient uses for PayForBlob submission.\n"+
			"0 disables queued submission. 1 uses the default account. Values greater than 1 automatically create "+
			"\"parallel-worker-*\" accounts and grant them fees using the default signer.",
```